aws/v20.0.1/release.yaml

```diff
@@ -0,0 +1,188 @@
+# Generated with:
+# /home/whites/workspace/devctl/devctl release create --base 20.0.0 --name 20.0.1 --overwrite --provider aws --component kubernetes@1.25.16 --app cluster-autoscaler@1.25.3-gs2 --app aws-ebs-csi-driver-servicemonitors@0.1.0
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  annotations:
+    giantswarm.io/docs: https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/
+  creationTimestamp: null
+  name: v20.0.1
+spec:
+  apps:
+  - componentVersion: 1.24.1
+    dependsOn:
+    - vertical-pod-autoscaler-crd
+    name: aws-cloud-controller-manager
+    version: 1.25.14-gs2
+  - componentVersion: 1.21.0
+    dependsOn:
+    - vertical-pod-autoscaler-crd
+    name: aws-ebs-csi-driver
+    version: 2.30.0
+  - dependsOn:
+    - prometheus-operator-crd
+    name: aws-ebs-csi-driver-servicemonitors
+    version: 0.1.0
+  - dependsOn:
+    - aws-cloud-controller-manager
+    - cilium
+    - coredns
+    - prometheus-operator-crd
+    - vertical-pod-autoscaler-crd
+    name: cert-exporter
+    version: 2.9.0
+  - componentVersion: 1.14.2
+    dependsOn:
+    - aws-cloud-controller-manager
+    - cilium
+    - coredns
+    - prometheus-operator-crd
+    - vertical-pod-autoscaler-crd
+    name: cert-manager
+    version: 3.7.4
+  - name: chart-operator
+    version: 3.2.1
+  - dependsOn:
+    - prometheus-operator-crd
+    name: chart-operator-extensions
+    version: 1.1.2
+  - componentVersion: 1.15.1
+    dependsOn:
+    - aws-cloud-controller-manager
+    name: cilium
+    version: 0.22.0
```

Comment:
hey @whites11 I think we are good to go with this upgrade. All the tests I ran show it's fine to skip 2 versions of cilium; so no network disruption during the upgrade. (that would be the case of updating directly to this release from a v19 release)

Reply:
ok, thanks!